server/app/models/va/Tripler.js

```diff
@@ -110,4 +110,8 @@ module.exports = {
   zip: "string",
   // The birth month (1-12) of this Tripler, according to the Ambassador.
   claimed_birth_month: "integer",
+  voted: {
+    type: "boolean",
+    default: true,
```

Comment:
Why is the default true? I think we assume someone hasn't voted until we hear otherwise.

Reply:
updated to false